notification/src/test/java/com/amazon/opendistroforelasticsearch/alerting/destination/CustomWebhookMessageTest.java

```diff
@@ -147,28 +170,28 @@ public void testCustomWebhookMessage_NonemptyEntityResponse() throws Exception {
         EasyMock.replay(httpResponse);
         EasyMock.replay(mockStatusLine);
 
-        DestinationHttpClient httpClient = new DestinationHttpClient();
-        httpClient.setHttpClient(mockHttpClient);
-        CustomWebhookDestinationFactory customDestinationFactory = new CustomWebhookDestinationFactory();
-        customDestinationFactory.setClient(httpClient);
+            DestinationHttpClient httpClient = new DestinationHttpClient();
+            httpClient.setHttpClient(mockHttpClient);
+            CustomWebhookDestinationFactory customDestinationFactory = new CustomWebhookDestinationFactory();
+            customDestinationFactory.setClient(httpClient);
 
-        DestinationFactoryProvider.setFactory(DestinationType.CUSTOMWEBHOOK, customDestinationFactory);
+            DestinationFactoryProvider.setFactory(DestinationType.CUSTOMWEBHOOK, customDestinationFactory);
 
-        Map<String, String> queryParams = new HashMap<String, String>();
-        queryParams.put("token", "R2x1UlN4ZHF8MXxxVFJpelJNVDgzdGNwMnVRenJwRFBHUkR0NlhROWhXOVVTZXpiTWx2azVr");
+            Map<String, String> queryParams = new HashMap<String, String>();
+            queryParams.put("token", "R2x1UlN4ZHF8MXxxVFJpelJNVDgzdGNwMnVRenJwRFBHUkR0NlhROWhXOVVTZXpiTWx2azVr");
 
         String message = "{\"Content\":\"Message gughjhjlkh Body emoji test: :) :+1: " +
                 "link test: http://sample.com email test: marymajor@example.com " +
                 "All member callout: @All All Present member callout: @Present\"}";
         BaseMessage bm = new CustomWebhookMessage.Builder("abc").withHost("hooks.chime.aws").
                 withPath("incomingwebhooks/383c0e2b-d028-44f4-8d38-696754bc4574").
-                withMessage(message).
+                withMessage(message).withMethod(method).
                 withQueryParams(queryParams).build();
         DestinationResponse actualCustomResponse = (DestinationResponse) Notification.publish(bm);
 
-        assertEquals(expectedCustomWebhookResponse.getResponseContent(), actualCustomResponse.getResponseContent());
-        assertEquals(expectedCustomWebhookResponse.getStatusCode(), actualCustomResponse.getStatusCode());
-    }
+            assertEquals(expectedCustomWebhookResponse.getResponseContent(), actualCustomResponse.getResponseContent());
+            assertEquals(expectedCustomWebhookResponse.getStatusCode(), actualCustomResponse.getStatusCode());
+        }
```

Comment:
Please remove the extra spacing, so the different code lines are on a consistent indent.

Reply:
Thanks! I believe I caught all the places where that somehow got changed.